iroh-ctl/Cargo.toml

```diff
@@ -2,10 +2,19 @@
 name = "iroh-ctl"
 version = "0.1.0"
 edition = "2021"
-authors = ["Kasey Huizinga <klhuizinga@gmail.com>"]
+authors = ["Kasey Huizinga <klhuizinga@gmail.com>", "Martijn Faassen <faassen@startifact.com>"]
 license = "Apache-2.0/MIT"
 repository = "https://github.com/n0-computer/iroh"
 description = "Client for interacting with running iroh processes."
+default-run = "iroh-ctl"
+
+[features]
+testing = []
+
+[[bin]]
+bench = false
```

Comment:
this shouldn't be necessary

Reply:
`bench=false` or the `[[bin]]` section? I had added the `[[bin]]` section so I could run the right command, but I realize now I did that in the prototype within `iroh`. It'll be necessary again once I move it into `iroh-one`.

I'll drop the `bench = false` bit; that's cargo culted and I copied it from somewhere.

Reply:
I meant just the bench false